src/styles/theme.js

```diff
@@ -1,5 +1,32 @@
 const theme = {
   colors: {
+    // dark blue as defined in Giant Swarm's style guide
+    darkBlue: '#234a61',
+
+    // Darker shades of above darkBlue.
+    // The higher the number, the darker the tone.
+    darkBlueDarker1: '#204358',
```

Comment:
What we need to have in mind right here, is what happens if we ever chang happa's colors? So what if we start using gray instead of blue?

I would much rather categorization based on color importance (primary / secondary / tertiary), and then splitting those by shade (100 - 900, 100 being the lightest, and 900 being the darkest)

Example
```js
{
  primary:  {
    100: '#fff',
    200: '#eee',
    300: '#ddd',
    400: '#ccc',
    500: '#bbb',
    600: '#aaa',
    700: '#999',
    800: '#888',
    900: '#777'
  }
}
```

Reply:
I'd rather use `rgba` or a helper function that outputs some `rgba` instead of harcoding the colors.

Reply:
You mean `hsl`, right?

Reply:
Having a specific set of shades will mean we'd have consistent colors across happa. Generating these colors might be done in a different way other than hardcoding them, but that shouldn't be done in the actual component styles, but in the top-level of the app.

Reply:
good point about not naming colors though!

Reply:
ok, yeah we can discuss this wednesday: not `hsl`we need alpha, so `hsla`or `rgba`

Reply:
Not sure about needing opacity for anything, since between those shades we're pretty much changing the lightness of the color, which is easier to do with `hsl`, rather than `rgb`/`rgba`

But I agree, we should talk about this in our meeting

Reply:
As interesting as that topic is, can we have it separate from this PR?